org.unofficialrevport.REVHubInterface.desktop

```
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=REV Hub Interface - Community Edition
+Exec=REVHubInterface
+Type=Application
+Icon=org.unofficialrevport.REVHubInterface.svg
```

```suggestion
Icon=org.unofficialrevport.REVHubInterface
```

it's the **name** not the file name.